text/0637-customizable-test-setups.md

```diff
@@ -4,7 +4,7 @@ start-date: 2020-06-01T00:00:00.000Z
 release-date: 2020-05-02T00:00:00.000Z
 release-versions:
   ember-source: v4.4.0
-  # ember-cli: FIXME
+  ember-cli: v4.3.0
```

Comment:
Implemented in https://github.com/ember-cli/ember-cli/pull/9803  and included in `ember new` as of 4.3.0 https://github.com/ember-cli/ember-new-output/commit/25b18e07b324f4a7e02c7ce53018a01e7575cc1e.